packages/graphql-connection-transformer/src/ModelConnectionTransformer.ts

```diff
@@ -485,11 +494,12 @@ export class ModelConnectionTransformer extends Transformer {
         // If the related type is not a list, the index has to be the default index and the fields provided must match the PK/SK of the index.
         if (!isListType(field.type)) {
             if (args.keyName) {
-                throw new InvalidDirectiveError('Connection is to a single object but the keyName provided does not reference the default table.')
+                throw new InvalidDirectiveError(`Connection is to a single object but the keyName ${args.keyName} ` +
```

Comment:
why are we concatenating still? if because of linter, there is a nicer way to fix it ;-)